### PR TITLE
Mention NIOPosix in README.

### PR DESCRIPTION
**Pull request by fabianfett**

### CODE REVIEW

**Review by Lukasa — COMMENTED** [review]
I'm afraid that down in the `Conceptual Overview` section there are a few places that say things like "provided by `NIO`" that need to be updated.

**Review by glbrntt — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -121,7 +122,7 @@ The basic I/O primitive of SwiftNIO is the event loop. The event loop is an obje
...
+In SwiftNIO today there is one [`EventLoopGroup`][elg] implementation, and two [`EventLoop`][el] implementations. For production applications there is the [`MultiThreadedEventLoopGroup`][mtelg], an [`EventLoopGroup`][elg] that creates a number of threads (using the POSIX [`pthreads`][pthreads] library) and places one `SelectableEventLoop` on each one. The `SelectableEventLoop` is an event loop that uses a selector (either [`kqueue`][kqueue] or [`epoll`][epoll] depending on the target system) to manage I/O events from file descriptors and to dispatch work. These [`EventLoop`s][el] and [`EventLoopGroup`s][elg] are provided by the `NIOCore` module. Additionally, there is the [`EmbeddedEventLoop`][eel], which is a dummy event loop that is used primarily for testing purposes, provided by the `NIOEmbedded` module.
```
Comment (glbrntt) [inline README.md:125]: This needs clarifying: ELG and EL are in `NIOCore` but SEL and MTELG are in `NIOPosix`

Comment (fabianfett) [reply README.md:125]: I think this paragraph is more about the implementations. Should just link to `NIOPosix`.

Comment (fabianfett) [reply README.md]: Changed to `NIOPosix`.

**Review by glbrntt — APPROVED** [review]

**Review by Lukasa — APPROVED** [review]
LGTM.

**Pull request merged**